.changeset/fifty-trains-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@segment/analytics-node': major
```

Comment:
Might want to coordinate with @MichaelGHSeg to add the breaking changes he needs to make for oauth so they both get included when we bump this (if you haven't coordinated already)

Reply:
What's the latest on that? We can make this a minor version bump since this is just a "deprecation", though it's pretty common to have deprecation as part of a major bump (and then removed in the next major bump).

Thoughts? I'd rather just get this in as a minor if it will be a while.